pkg/sql/authorization.go

```diff
@@ -123,9 +123,6 @@ func (p *planner) CheckPrivilege(
 }
 
 // IsOwner returns if the role has ownership on the descriptor.
-// TODO(richardjcai): Add checks for if the user owns the parent of the object.
```

Comment:
did you did this in this PR?

Reply:
No but this comment is inaccurate so I removed this. We're not actually supposed to do this.